hyperspy/tests/misc/test_cython.py

```diff
@@ -0,0 +1,22 @@
+import nose.tools as nt
+import warnings
+
+import os
+
+my_path = os.path.dirname(__file__)
+
+def test_cythonization():
+    try:
+        import Cython
+    except ImportError:
```

Comment:
Nice, I didn't knew that, I was wondering how to do it (I sow some skipped tests in Travis CI conclusions) I am changing it right now.